docs/guides/2-cli.md

```diff
@@ -60,6 +61,86 @@ Here you can build a [custom ruleset](../getting-started/3-rulesets.md), or exte
 - [OpenAPI ruleset](../reference/openapi-rules.md)
 - [AsyncAPI ruleset](../reference/asyncapi-rules.md)
 
+## Scoring the API
+
+Scoring an API definition is a way to understand in a high level, how compliant is the API definition with the rulesets provided. This helps teams to understand the quality of the APIs regarding the definition.
+
+The scoring is produced in two different metrics:
+
+- A number scoring. Who cames as substracting from 100% from any error or warning
+- A letter, who groups numeric scorings in letters from A (better) to any
```

Comment:
Suggest:

A letter scoring, which groups numeric scoring in letters from A to Z, with A being the best score.